.github/workflows/nightly-wheels.yml

```
@@ -30,6 +36,7 @@ jobs:
           - "3.10"
           - "3.11"
           - "3.12"
+          - "3.14"
```

just wondering if it works
```suggestion
          - "3.14.0-rc.1"
```